server/api/crud/runtimes/nuclio/function.py

```diff
@@ -183,6 +184,37 @@ def _compile_function_config(
     # resolve env vars before compiling the nuclio spec, as we need to set them in the spec
     env_dict, external_source_env_dict = _resolve_env_vars(function)
 
+    project = function.metadata.project or mlrun.mlconf.default_project
+    tag = function.metadata.tag
+
+    serving_spec_volume = None
+    serving_spec = function._get_serving_spec()
+    if serving_spec is not None:
+        if len(serving_spec) >= mlrun.mlconf.httpdb.nuclio.serving_spec_env_cutoff:
```

Comment:
Add a comment explaining why we are doing this

Reply:
I meant to comment the "why" instead of the "what" - something like:

since environment variables have a limited size, large serving specs are stored in config maps that are mounted to the pod